students/JerryH/Lesson03/list_lab.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+
+fruits = ['Apples', 'Pears', 'Oranges', 'Peaches']
+#Series 1
+def series1():
+    print(fruits)
+    addFruit = input("What fruit do you want to add?\n")
+    fruits.append(addFruit)
+    print(fruits)
+    num = int(input("Enter a number:\n"))
+    print(num)
+    print("The fruit is {}".format(fruits[num-1]))
+    someFruit = input("Add another fruit:\n")
+    fruits = [someFruit] + fruits
+    print(fruits)
+    anotherFruit = input("Add another fruit again:\n")
+    fruits.insert(0,anotherFruit)
+    print(fruits)
+    for i in range(len(fruits)):
+        if fruits[i][0] == 'P':
+            print(fruits[i], end=" ")
+
+#Series 2
+def series2():
+    print(fruits)
+    fruits.pop()
+    print(fruits)
+    toDelete = input("Which fruit do you want to delete? \n")
+    for f in fruits:
+        if f == toDelete:
+            fruits.remove(toDelete)
+    print(fruits)
+
+#Series #3
+def series3():
+    print([f.lower() for f in fruits])
+    ans = input("Do you like apples?\n")
+    ans = ans.lower()
+    while ans != 'yes' and ans != 'no':
+        ans = input("Please answer yes or no.\n")
+
+    if ans == 'no':
+        fruits.remove('Apples')
+    print(fruits)
+
+#Series #4
+def series4():
+    new_fruits = fruits
+    for i in range(len(new_fruits)):
+        new_fruits[i] = new_fruits[i][-1::-1]
+    fruits.pop()
+    print(new_fruits)
+    print(fruits)
```

Comment:
for this and other modifications keep in mind that making a copy like:
```
new_list = old_list
```
does a shallow copy, meaning that any changes you make to `new_list` will make changes to `old_list`

you can use either of the methods below to make deepcopy
```
>>> new_fruits = fruits[:]
>>> from copy import deepcopy
>>> new_fruits = deepcopy(fruits)
```